### ISSUE TITLE
Generic (Note) Function

### ISSUE THREAD

**Issue by tosokuzma2020** [dsifford/academic-bloggers-toolkit#603]
Hello there, 
I'm a big fan of your plugin and love the implementation! Really happy with the interface, the reliability and the fact that footnotes don't just disappear even when the plugin is deactivated.

Having said that, I'm really having some trouble with the disappearance of the generic (note function). First of all, the issue is that the formatting of the regular Wordpress footnotes is different than for the Academic Toolkit citations (I can't seem to integrate numbering into footnotes, the footnotes and citations don't appear at the same exact spot etc.). Secondly, I sometimes want to include certain citations with the plugin, and then some unformatted footnotes (say text) - but now I can't do that anymore because the footnotes and the citations would be ordered separately (so that I always have to decide whether I want to use the one or the other). Lastly, I can't seem to access the old citations I included with the generic note function anymore or reinsert them precisely because there is no such function anymore.

Is there ANY hope that the generic (note) function for manual citations might reappear? 
Thank You and all the best.
toso

**Comment by tosokuzma2020** [comment]
PS Just saw that - if I understood correctly - it was completely intended to handle generic notes and citations separately, so congratulations on that! Perhaps there would still be a way of adding a generic note function to the citation interface as well (so that one could chose if one inserts footnote-like content via footnotes or as citations)? Basically, otherwise I see no way of say referring to several items at once, or of commenting upon the items, as one often does when citing literature.

**Comment by dsifford** [comment]
You should be able to do this by clicking "add manually" from the insert citation dialog and select "generic (note)".

**Comment by tosokuzma2020** [comment]
Hi, thank you for taking your time to respond! That is indeed how it used to work, but if I am not mistaken the "generic (note)" option has disappeared from newer versions of the plugin! I just double-checked and when I want to manually insert a citation in the current version, there is no such option (as there was in previous versions when there was not yet a footnote function).

**Comment by dsifford** [comment]
Ah, my mistake. 

Yes I deleted that option because it became too difficult to deal with notes interspersed in citations. That was the motivation to support footnotes.

For now, I have no intention (or bandwidth) to support both styles. Sorry about that.

**Comment by tosokuzma2020** [comment]
Understood, thanks anyway! Like someone else who has posted here, for me a way to choose whether Footnotes are numbered or symbolized would also work, but I understand if that too, is too much work. 

At any rate, I've found a temporary work-around which is not perfect but as I noted in my first comment, the fact that your plugin works without shortcodes (and allows footnotes to be shown even when it is uninstalled/deactivated) just makes it so much better than all the other options I've found!
